Component Plug-in (CP) Examples/README.md

```diff
@@ -1,6 +1,6 @@
 ## How to install
 * Compress all files/directories under the example of your choice into a zip
-* E.g. `zip mapField.zip mapField/mapField mapField/appian-component-plugin.xml`
+* E.g. `zip -r mapField.zip mapField/mapField mapField/appian-component-plugin.xml`
```

Comment:
I thought it was `-R` for the recursive option, but maybe I'm wrong

Reply:
Looked up to confirm and it's `-r`: https://linux.die.net/man/1/zip

Also verified by executing with `-R` on local and it does not recurse through the `mapField` directory while `-r` does.

Reply:
Ah, `-R` is also recursive but uses the local directory & does pattern matching, ignore my comment